meson_options.txt

```diff
@@ -13,6 +13,7 @@ option('cdda', type: 'feature', value: 'disabled', description: 'cdda support (l
 option('cplugins', type: 'feature', value: 'auto', description: 'C plugins')
 option('dvbin', type: 'feature', value: 'disabled', description: 'DVB input module')
 option('dvdnav', type: 'feature', value: 'auto', description: 'dvdnav support')
+option('gnome', type: 'feature', value: 'auto', description: 'gnome inhibit support')
```

Comment:
and what about waf? Your stated goal of servicing stable ubuntu with GNOME won't work if you rely on the very recent meson that mpv requires